### PR TITLE
Ali/TempPR/unittest2

### PR DESCRIPTION
**Pull request by m-ali-0001**
### Description

[LP-](https://philanthropyu.atlassian.net/browse/LP-)

[philu-edx-theme PR](https://github.com/philanthropy-u/philu-edx-theme/pull/###)

Add a description of your changes here.

### CODE REVIEW

**Comment by m-ali-0001** [comment]
jenkins run python

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run python

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by zeshanarifios** [comment]
Let's Please Update the title of PR containing the purpose of this temporary PR with [Don't Merge] initial if it is only for testing purpose. Thank You!
@aliassad

File: lms/envs/test.py [hunk 1]
```diff
@@ -656,3 +656,33 @@
...
+
+ENV_TOKENS = dict()
+
+if ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR'):
+    COMPREHENSIVE_THEME_DIR = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR')
+
+COMPREHENSIVE_THEME_DIRS = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIRS', COMPREHENSIVE_THEME_DIRS) or []
+COMPREHENSIVE_THEME_LOCALE_PATHS = ENV_TOKENS.get('COMPREHENSIVE_THEME_LOCALE_PATHS', [])
+
+NODEBB_MASTER_TOKEN = "1dae9ee7-3b32-4d17-a35b-c179a1788ec3"
```
Comment (aliassad) [inline lms/envs/test.py:684]: @m-ali-0001  can we put these tokens some how in any environment file eg. test_env ?

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Pull request closed without merging**